k8s/plugin/README.md

```diff
@@ -338,3 +338,85 @@ make Rest, Loki and etcd service available outside the cluster. One way to acces
   kubectl port-forward service/mayastor-etcd 2379:2379 -n mayastor
   ```
 </details>
+
+
+<details>
+<summary> Upgrade operations </summary>
+
+```sh
+kubectl-mayastor-install
+`Install` upgrade operator
```

Comment:
Don't think you need to show all of these, but I see that the dump command started showing all of these where the previous commands didn't.